docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,427 @@
+# Sticker
+
+
+The Sticker feature provides users with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and [Sticker consumer application](#consumer_application). The Sticker provider creates stickers and developers can create their own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.
+
+The main features of the Sticker API include:
+
+- Preparing the Sticker service for use
+
+  You can connect the Sticker daemon to operate the Sticker service.
+
+- Sticker data handle
+
+  You can use the Sticker data handle to [store the sticker information](#set_data_handle) and to [get the sticker information](#get_data_handle) stored in it.
+
+- [Inserting the sticker information](#insert_sticker)
+
+  - You can store the sticker information using the Sticker API at application runtime, and [use the JSON file](#set_json) to store the sticker information at a time.
+  - You can store sticker information automatically when the Sticker provider application is installed without using the Sticker API.
+
+- Retrieving the sticker information
+
+  You can [retrieve sticker information](#retrieve_sticker), which includes URI, keyword, group name, and so on.
+
+**Figure: Sticker Mechanism**
+
+![Sticker Mechanism](./media/sticker.png)
+
+## Prerequisites
+
+To enable your Sticker provider application to use the Sticker functionality:
+
+1. To use the Sticker provider API, the following category must be added to the `tizen-manifest.xml` file:
+
+    ```
+    <category name="http://tizen.org/category/sticker"/>
+    ```
+
+2. To use the functions and data types of the Sticker API, include the `<sticker_provider.h>` header file in your application:
+
+    ```
+    #include <sticker_provider.h>
+    ```
+
+To enable your Sticker consumer application to use the Sticker functionality:
+
+1. To use the Sticker consumer API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/mediastorage</privilege>
+    </privileges>
+    ```
+
+2. To use the functions and data types of the Sticker API, include the `<sticker_consumer.h>` header file in your application:
+
+    ```
+    #include <sticker_consumer.h>
+    ```
+
+ To ensure that a Sticker function has been executed properly, make sure that the return is equal to `STICKER_ERROR_NONE`.
+
+<a name="provider_application"></a>
+## Creating Sticker Provider Application
```

Comment:
Global: Please change all headings to verb form as mentioned by @mijin85cho in the https://github.com/Samsung/tizen-docs/pull/884#issuecomment-546825940 comment:

Hello, @Inhong
Would you please check the comments?

Also, adding to the above comments,

please use verb form to the headings.

ex ) Creating Sticker Provider Application -> Create Sticker Provider Application

Reply:
Fixed.